### DISCUSSION TITLE
where the sales price format

### DISCUSSION THREAD

**Discussion by AlonsoMartinezh** [openstore-ecommerce/OpenStore#188] [General]
where the sales price format is modified to add the thousands separator

**Comment by DNNMonster** [comment]
If I understand the question correctly it sounds like you want to create custom templates.  Most likely you would be creating a ProductDisplayDetail.cshtml and a ProductDisplayListBody.cshtml since they are doing the rendering for your product data to your end users.  You can format the price or do anything you like once you've got you're custom templates in the right directory.   

There is some pretty good info on Themes and templates in the documentation about half way down the page.

https://doc.openstore-ecommerce.com/

**Reply by AlonsoMartinezh** [reply]
I appreciate your valuable collaboration, I will proceed to consultations how to modify the templates, thank you
